src/main/containers/MockupsPanel.jsx

```diff
@@ -72,6 +74,10 @@ const MockupsPanel = (props) => {
     setCreateTestIndex(value);
   }
 
+  function saveFile(e) {
+    ipcRenderer.send('save_file', JSON.stringify({ data, tests }));
```

Comment:
Found a potential bug here 🐞

### Steps to reproduce:
1. Get source data (I used https://jsonplaceholder.typicode.com/users)
2. Click **Save Work as JSON**
3. Save as “Untitled” to a path where “Untitled” already exists
4. A dialog should appear asking if you want to replace the file
5. If you choose to replace, you should get an infinite error loop—must force quit app to get rid of it.

The error is: `{"code": "EISDIR","errno":-21,"path":"/<file path>","syscall":"open"}`

Again, this caused an **infinite error loop** on my machine—there was no way to get rid of it or use the app until I force quit the process.